specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2264 @@
+        "name",
+        "centerFrequencyMHz",
+        "bandwidthMHz",
+        "direction",
+        "polarization"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink).",
+          "enum": [
+            "uplink",
+            "downlink"
+          ],
+          "x-ms-enum": {
+            "name": "direction",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "authorizations": {
+          "type": "array",
+          "description": "Authorized Ground Stations",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AuthorizedGroundstation"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "List of authorized spacecraft links per ground station and the expiration date of the authorization."
+    },
+    "Spacecraft": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SpacecraftsProperties",
+          "description": "Spacecraft Properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SpacecraftsProperties"
+            }
+          ]
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Customer creates a spacecraft resource to schedule a contact."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
I don't have strong feelings on this. @rkmanda is there any guidance of this you know of?

Reply:
I think most commonly its modelled as *not* required, so that the payload model can be reused for PATCH scenario,  for things like update tags, update resource identity, etc.